base_synchro/models/base_synchro_obj.py

```diff
@@ -96,10 +101,15 @@ class BaseSynchroObjLine(models.Model):
     _description = "Synchronized instances"
 
     name = fields.Datetime(
-        string="Date",
+        "Date",
         required=True,
-        default=lambda *args: time.strftime("%Y-%m-%d %H:%M:%S"),
+        default=lambda *args: time.strftime(DEFAULT_SERVER_DATETIME_FORMAT),
```

Comment:
Use `fields.Datetime.to_string(datetime.date.today()` instead of `time.strftime(DEFAULT_SERVER_DATETIME_FORMAT)`